specification/nhs-website-content-api.yaml

```diff
@@ -259,12 +228,13 @@ paths:
                 $ref: components/examples/Medicines.json
   /medicines/*:
```

Comment:
Should the root 'Medicines A to Z' (/medicines/) endpoint be mentioned here too?

Reply:
The root page is above